test-framework/Vagrantfile

```diff
@@ -38,6 +39,26 @@ if ENV['GIT_BRANCH'].nil?
 end
 
 Vagrant.configure("2") do |config|
+  # Note: This requires Vagrant 2.2.0 or newer
+  config.trigger.before :up do |trigger|
+    trigger.info = "Acquiring Lock..."
+    trigger.ruby do |env, machine|
+      # The vagrant-libvirt plugin has a race condition getting
```

Comment:
is this documented anywhere, or just an observation?  If it's an actual bug someplace, a reference here might be nice.

Reply:
An observation, which I haven't seen in the Issue Tracker for vagrant-libvirt. Running multiple vagrant-libvirt vagrant instances at the same time doesn't seem to be a common use-case for the plugin.